node_modules/col-core/lib/server.js

```diff
@@ -0,0 +1,257 @@
+/**
+ * Copyright 2015 UC Berkeley (UCB) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('lodash');
+var bodyParser = require('body-parser');
+var config = require('config');
+var express = require('express');
+var http = require('http');
+var util = require('util');
+
+var log = require('./logger')('col-core/server');
+
+var safePathPrefixes = [];
+
+/**
+ * Start the Collabosphere Express server on the configured port
+ *
+ * @return {Express}                The created express server
+ */
+var setUpServer = module.exports.setUpServer = function() {
+  // Create the express server
+  var app = express();
+
+  // Expose the HTTP server on the express app to allow other modules to hook into it
+  app.httpServer = http.createServer(app);
+
+  // Start listening for requests
+  var port = config.get('app.port');
+  app.httpServer.listen(port, 'localhost');
+
+  // Don't output pretty JSON
+  app.set('json spaces', 0);
+
+  // Don't output the x-powered-by header
+  app.set('x-powered-by', false);
+
+  /*!
+   * Collabosphere supports the following type of request encodings:
+   *
+   *  * urlencoded (regular POST requests)
+   *  * application/json
```

Comment:
Is there a need for a JSON or multipart parser? It seems like most of the API interactions with this tool should be doable through plain-old urlencoded requests right?